### ISSUE TITLE
Solution doesn't build anymore

### ISSUE THREAD

**Issue by dtriana** [LogicAppUnit/TestingFramework#40]
Not sure if related to the updates from .netconf/.net9 but the solution no longer builds

Output errors

Error (active)	NU1902	Warning As Error: Package 'Microsoft.AspNetCore.Server.IIS' 2.2.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-prrf-397v-83xh	LogicAppUnit	G:\Github\TestingFramework\src\LogicAppUnit\LogicAppUnit.csproj	1	

Error (active)	NU1904	Warning As Error: Package 'System.Text.Encodings.Web' 4.6.0 has a known critical severity vulnerability, https://github.com/advisories/GHSA-ghhp-997w-qr28	LogicAppUnit.Samples.Functions	G:\Github\TestingFramework\src\LogicAppUnit.Samples.Functions\LogicAppUnit.Samples.Functions.csproj	1	

Error (active)	NU1902	Warning As Error: Package 'Microsoft.AspNetCore.Server.IIS' 2.2.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-prrf-397v-83xh	LogicAppUnit.Samples.LogicApps.Tests	G:\Github\TestingFramework\src\LogicAppUnit.Samples.LogicApps.Tests\LogicAppUnit.Samples.LogicApps.Tests.csproj	1	

As a workaround, updating all

```<TreatWarningsAsErrors>True</TreatWarningsAsErrors>```

to false allowed the solution to build

**Comment by mark-abrams** [comment]
Thats strange, the solution builds ok on my laptop and on my Azure Devops build environment.  Are you able to share the full build output please?

**Comment by dtriana** [comment]
Yup, definitely strange, it was working fine until the latest updates released as part of dotnet conf,

`git clone https://github.com/LogicAppUnit/TestingFramework.git
Cloning into 'TestingFramework'...
remote: Enumerating objects: 672, done.
remote: Counting objects: 100% (243/243), done.
remote: Compressing objects: 100% (129/129), done.
remote: Total 672 (delta 192), reused 115 (delta 112), pack-reused 429 (from 1)
Receiving objects: 100% (672/672), 294.50 KiB | 1.63 MiB/s, done.
Resolving deltas: 100% (374/374), done.
 anothernewclone  ls

    Directory: G:\Github\anothernewclone

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----          11/18/2024  8:12 AM                TestingFramework

 anothernewclone  cd .\TestingFramework\src
 src  dotnet build
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit\LogicAppUnit.csproj : error NU1902: Warning As Error: Package 'Microsoft.AspNetCore.Server.IIS' 2.2.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-prrf-397v-83xh
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit.Samples.LogicApps.Tests\LogicAppUnit.Samples.LogicApps.Tests.csproj : error NU1902: Warning As Error: Package 'Microsoft.AspNetCore.Server.IIS' 2.2.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-prrf-397v-83xh
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit\LogicAppUnit.csproj : error NU1903: Warning As Error: Package 'System.Net.Http' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-7jgj-8wvc-jh57
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit.Samples.LogicApps.Tests\LogicAppUnit.Samples.LogicApps.Tests.csproj : error NU1903: Warning As Error: Package 'System.Net.Http' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-7jgj-8wvc-jh57
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit\LogicAppUnit.csproj : error NU1903: Warning As Error: Package 'System.Text.Json' 6.0.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-8g4q-xg66-9fp4
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit.Samples.LogicApps.Tests\LogicAppUnit.Samples.LogicApps.Tests.csproj : error NU1903: Warning As Error: Package 'System.Text.Json' 6.0.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-8g4q-xg66-9fp4
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit\LogicAppUnit.csproj : error NU1903: Warning As Error: Package 'System.Text.RegularExpressions' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-cmhx-cq75-c4mj
    G:\Github\anothernewclone\TestingFramework\src\LogicAppUnit.Samples.LogicApps.Tests\LogicAppUnit.Samples.LogicApps.Tests.csproj : error NU1903: Warning As Error: Package 'System.Text.RegularExpressions' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-cmhx-cq75-c4mj

Restore failed with 8 error(s) in 1.3ss`

**Comment by mark-abrams** [comment]
@dtriana , version 1.11.0 has been released to NuGet that fixes this issue.
